src/redis.ts

```diff
@@ -0,0 +1,1256 @@
+    count?: number | undefined
+  ) {
+    const command: string = "SSCAN";
+
+    let patternArray: any[] = [];
+    if (pattern !== undefined) {
+      patternArray.push("MATCH");
+      patternArray.push(pattern);
+    }
+
+    let countArray: any[] = [];
+    if (count !== undefined) {
+      countArray.push("COUNT");
+      countArray.push(count);
+    }
+
+    return this._commandParser(
+      command,
+      collection,
+      key,
+      cursor,
+      ...patternArray,
+      ...countArray
+    );
+  }
+
+  sunion(keys: string[], collection: string) {
+    const command: string = "SUNION";
+    return this._commandParser(command, collection, ...keys);
+  }
+
+  sunionstore(destination: string, keys: string[], collection: string) {
+    const command: string = "SUNIONSTORE";
+    return this._commandParser(command, collection, destination, ...keys);
+  }
+  // End of SET commands
+
+  // Start of SORTED SET commands
+  zadd(key: string, data: any[], collection: string, options: any[] = []) {
+    const command: string = "ZADD";
+    return this._commandParser(command, collection, key, ...options, ...data);
+  }
+
+  zcard(key: string, collection: string) {
+    const command: string = "ZCARD";
+    return this._commandParser(command, collection, key);
+  }
+
+  zcount(key: string, minimum: string, maximum: string, collection: string) {
+    const command: string = "ZCOUNT";
+    return this._commandParser(command, collection, key, minimum, maximum);
+  }
+
+  zdiff(
+    numKeys: number,
+    keys: string[],
+    collection: string,
+    withScores: boolean = false
+  ) {
+    const command: string = "ZDIFF";
+    let withScoresCommand: string | undefined;
+    if (withScores) {
+      withScoresCommand = "WITHSCORES";
+    } else {
+      withScoresCommand = undefined;
+    }
+    return this._commandParser(
+      command,
+      collection,
+      numKeys,
+      ...keys,
+      withScoresCommand
+    );
+  }
+
+  zdiffstore(
+    destination: string,
+    numkeys: number,
+    keys: string[],
+    collection: string
+  ) {
+    const command: string = "ZDIFFSTORE";
+    return this._commandParser(
+      command,
+      collection,
+      destination,
+      numkeys,
+      ...keys
+    );
+  }
+
+  zincrby(key: string, increment: number, member: string, collection: string) {
+    const command: string = "ZINCRBY";
+    return this._commandParser(command, collection, key, increment, member);
+  }
+
+  zinter(
+    numKeys: number,
+    keys: string[],
+    collection: string,
+    options: any[] | undefined = undefined,
+    withScores: boolean = false
+  ) {
+    const command: string = "ZINTER";
+
+    let optionsArray: any[] = [];
+    if (options !== undefined) {
+      optionsArray = [...options];
+    }
+    if (withScores) {
+      optionsArray.push("WITHSCORES");
+    }
+
+    return this._commandParser(
+      command,
+      collection,
+      numKeys,
+      ...keys,
+      ...optionsArray
+    );
```

Comment:
common lines can be moved to a function and can be used at multiple places. Please check all the similar instances.